docker-entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+set -e
+
+. /opt/pysetup/.venv/bin/activate
+
+exec python3 -m clairvoyance "$@"
```

Comment:
I wonder why do we need `exec` here 🤔

Reply:
It's a docker practice allowing to forward everything (signals, env, ...) to the new shell.

You can read more about this:
https://stackoverflow.com/questions/39082768/what-does-set-e-and-exec-do-for-docker-entrypoint-scripts